### PR TITLE
[js] Add release notes for 1.53.1

### PR DESCRIPTION
**Pull request by jjtang1985**
## What Has Changed?

This PR is re-created, as the [old PR](https://github.com/SAP/cloud-sdk/pull/640) failed due to magic vale issue. 
The vale issue was also reported by others [here](https://github.com/errata-ai/vale-action/issues/48)

## Manual Checks?

- [ ] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [ ] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [ ] You formatted all changed files with prettier (`npm run prettier`)
- [ ] You tested if the documentation still builds (`npm run build`)
- [ ] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [ ] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

**Comment by jjtang1985** [comment]
@MattKuhr , we faced the same vale issue again, and only recreation can fix the it.
Please feel free to vote for [this issue](https://github.com/errata-ai/vale-action/issues/48), if you like.

File: docs/js/release-notes.mdx [hunk 1]
```diff
@@ -30,15 +30,15 @@ It will help you:
...
+For your convenience we [pregenerated client libraries](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*) (also known as Virtual Data Model or VDM for short) for allow-listed OData services of SAP S/4HANA Cloud, Marketing Cloud, and SAP S/4HANA On-premise.
 These libraries are not Open Source, but free and distributed under the SAP Developer license through [npmjs.com](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*).
 You can always generate a typed client library yourself by using our [OData](features/odata/generate-odata-client.mdx) and [Open API](features/openapi/generate-openapi-client.mdx) code generators.
 To easily search for services and get typed client library for them, use our [API Business Hub integration](https://blogs.sap.com/2021/09/17/the-sap-cloud-sdk-integrates-with-the-sap-api-business-hub/).
 
 :::
 
 <!-- Copy and fill
-## 1.xx.x [Core Modules] - January 1, 2021
+## 1.xx.x [Core Modules] - January 1, 2022
```
Comment (marikaner) [inline docs/js/release-notes.mdx:41]:
```suggestion
## 1.xx.x [Core Modules] - Jan 1, 2022
```

Comment (marikaner) [inline docs/js/release-notes.mdx:41]:
```suggestion
## 1.xx.x [Core Modules] - Jan 1, 2022
```

**Review by marikaner — APPROVED** [review]
Let's have the discussion you suggested, but in the meantime I think this can already be merged and then improved after the discussion.

**Comment by artemkovalyov** [comment]
This is a PR with an alternative strategy to provide release notes for a product as a whole: https://github.com/SAP/cloud-sdk/pull/642.

Let's have it as a part of the discussion initiated by @jjtang1985

**Comment by artemkovalyov** [comment]
In the meanwhile, we can merge this.

**Pull request merged**